authfile.c

```diff
@@ -442,9 +442,7 @@ sshkey_in_file(struct sshkey *key, const char *filename, int strict_type,
 int
 sshkey_check_revoked(struct sshkey *key, const char *revoked_keys_file)
 {
-	int r;
-
-	r = ssh_krl_file_contains_key(revoked_keys_file, key);
+	int r = ssh_krl_file_contains_key(revoked_keys_file, key);
```

Comment:
style guide (https://man.openbsd.org/style.9) says not to use function calls as variable initializers.